README.md

```diff
@@ -333,6 +333,16 @@ When `enableLowInitialPlaylist` is set to true, it will be used to select
 the lowest bitrate playlist initially.  This helps to decrease playback start time.
 This setting is `false` by default.
 
+##### ignorePlayerSize
+* Type: `boolean`
+* can be used as an initialization option
+
+When `ignorePlayerSize` is set to true, selection logic will ignore the
+player size and rendition resolutions when making a decision.  This should
```

Comment:
The should part here is a bit unclear to me, but I may not fully understand the scenario. Any more details on that part?

Reply:
I am opening this PR because I had a customer using our per-title encoding solution which is marketed at https://mux.com/per-title-encoding/ and described in all kinds of fun technical detail at - https://mux.com/blog/per-title-encoding-scale/ - a specific video was seen to never play anything "reasonably good looking" given the generated rendition set which I put in the unit tests. most people in the industry would look at that ladder and say "what?" so I wrote this comment. but since the comment had a "what?" response, I guess I'll delete it.  😄